packages/seacas/libraries/supes/ext_lib/exname.c

```diff
@@ -71,7 +71,15 @@ void exname(FTNINT *iunit, char *name, FTNINT *ln, long int nlen)
 
   if (*iunit > -100 && *iunit < 100) {
 #if Build64
-    snprintf(string, 3, "%02ld", labs(*iunit));
+    //To remove the warning for "snprintf(string, 3, "%02ld", labs(*iunit));" with minimal performance changes
```

Comment:
I think the least distruptive fix is
```
int ret = snprintf(string, 3, "%02ld", labs(*iunit));
if (ret < 0) {
... issue an exodus error and exit the code
}
```

Reply:
This should never happen since iunit is verified to be 2 digits or less and it silences the warning.